Header header
int32 num
bool is_detected

